jina/types/document/__init__.py

```diff
@@ -1006,15 +1006,25 @@ def convert_image_buffer_to_blob(self, color_axis: int = -1):
         """
         self.blob = to_image_blob(io.BytesIO(self.buffer), color_axis)
 
+    @deprecated_parameter(params=['width', 'height'])
```

Comment:
remove it

Reply:
Why? This may be used by people in executors and solutions